homeassistant/components/zwave_js/sensor.py

```diff
@@ -781,7 +790,7 @@ def __init__(
             additional_info=[property_key_name] if property_key_name else None,
         )
 
-    @property
+    @property  # type: ignore[override]
```

Comment:
These probably can be cached, but that is left up to the zwave_js code owners for a future PR as I didn't want to make a change here that might have side effects.